src/realmCryo.ttl

```diff
@@ -50,57 +52,133 @@ sorelsp:inside rdf:type owl:ObjectProperty .
 ###  http://sweetontology.net/realmCryo/AblationZone
 soreac:AblationZone rdf:type owl:Class ;
                   rdfs:subClassOf soreac:GlacialRegion ;
-                  rdfs:label "ablation zone"@en .
+                  rdfs:label "ablation zone"@en ;
+                  skos:closeMatch <http://purl.obolibrary.org/obo/ENVO_01000914> ;
+                  skos:definition  [
+                   rdfs:comment  "An environmental zone which overlaps a glacier or an ice sheet, and in which ice loss exceeds ice gain."@en ;
```

Comment:
Trying to be consistent with ENVO definition which is worded exactly the same.

Reply:
I'd assert the suggetion is consistent with envo :
envo environmental zone: "A site which has its extent determined by the presence or influence of one or more components of an environmental system or the processes occurring therein. "
overlap: (not defined in ENVO) how about Merriam-Webster "to extend over or past and cover a part of"  
component of environmental system --> Glacier or ice sheet
process -- ice loss exceeds ice gain
overlap-- 'a part of' (unless the ablation zone doesn't necessarily have to be part of a glacier or ice sheet--is that the case??)

If SWEET is going to 'inherit' meaning from ENVO terms then 'environmental zone' in the definitions should be linked to the ENVO '[environmental zone concept](http://purl.obolibrary.org/obo/ENVO_01000408)'

Reply:
Collective comments from our [Semantic Harmonization call](
https://docs.google.com/document/d/1IFA6AsCdxESh9mZB5W1bZ_vARFkF3OvnL-KgdGNcp60/edit#heading=h.2uotl4egsd38)

The semantics of zones (as BFO:sites) in ENVO may be changed in cases like "ablation zone" as the difference between the spatial entity and the physical entity is unlikely to have much substance (unlike protected areas or sacred sites). 

The ENVO IRI mapped here will remain consistent.

Agreement that the textual definition (if maintained) should use parthood rather than overlap relations to constrain the asserted extent of the zone.

Reply:
Note the ideas of [part of](http://purl.obolibrary.org/obo/BFO_0000050) and [overlap ](http://purl.obolibrary.org/obo/RO_0002131)in ENVO are coming from RO and BFO.

Note that these spatial relations have emerged from biology - a variation of these that are native to Earth and space science are needed, aligned with [GEOSPARQL conventions](https://www.ogc.org/standards/geosparql).

Reply:
this is an ENVO definition,